server/models/meteorological.py

```diff
@@ -1,18 +1,20 @@
-from sqlalchemy import Column, Float, Integer, String
+from sqlalchemy import Column, DateTime, Float, ForeignKey, Integer, String
 
 from server.db.base_class import Base
 
 
-class Historical_Meteorological(Base):
-    __tablename__ = "Historical Meteorological Data"
+class Meteorological(Base):
+    __tablename__ = "Meteorological"
 
     id = Column(Integer, index=True, nullable=False, primary_key=True)
-    station_id = Column(String(10), index=True, nullable=False, unique=False)
-    year = Column(String, nullable=False)
-    month = Column(String, nullable=False)
-    day = Column(String, nullable=False)
-    hour = Column(String, nullable=False)
-    minute = Column(String, nullable=False)
+    station_id = Column(
+        String(10),
+        ForeignKey("buoy.station_id"),
+        index=True,
+        nullable=False,
+        unique=False,
+    )
+    datetime = Column(DateTime, nullable=False)
```

Comment:
Rename it to `recorded_on: datetime` or something like that.

It's best to rename the attribute on line 10 because it shares the name with Python's standard library `datetime`.

And I like and appreciate how you were consistent with this field across all your models. Once you make this change, let's continue that pattern you set by updating the models and schemas you added.